sql/src/main/java/io/crate/operation/reference/information/InformationTablePartitionsExpression.java

```diff
@@ -30,6 +30,8 @@
 abstract class InformationTablePartitionsExpression<T>
     extends RowContextCollectorExpression<PartitionInfo, T> {
 
+    public static final BytesRef tableType = new BytesRef("BASE TABLE");
```

Comment:
ah, I see it's also used down there. https://github.com/crate/crate/pull/5641/files#diff-b98d4715cbfe1b1d080ebe91ea61756aR62